packages/core/src/external/saml/__tests__/constants.ts

```diff
@@ -153,46 +144,31 @@ const iti39BodyData = {
   ],
 };
 
-const privateKey = getEnvVarOrFail("IHE_STAGING_KEY");
-const publicCert = getEnvVarOrFail("IHE_STAGING_CERT");
-
-describe("SAML Signature Verification", () => {
-  it("should sign and verify the XCPD SOAP envelope successfully", () => {
-    const soapEnvelope = createITI5SoapEnvelope({
-      bodyData: iti55BodyData,
-      publicCert: publicCert,
-    });
-
-    const signedWithTimestamp = signTimestamp({ xml: soapEnvelope, privateKey });
-    expect(verifySaml({ xmlString: signedWithTimestamp, publicCert: publicCert })).toBeTruthy();
-
-    const signedEnvelope = signEnvelope({ xml: signedWithTimestamp, privateKey });
-    expect(verifySaml({ xmlString: signedEnvelope, publicCert: publicCert })).toBeTruthy();
-  });
-
-  it("should sign and verify the ITI38 envelope successfully", () => {
-    const soapEnvelope = createITI38SoapEnvelope({
-      bodyData: iti38BodyData,
-      publicCert: publicCert,
-    });
-
-    const signedWithTimestamp = signTimestamp({ xml: soapEnvelope, privateKey });
-    expect(verifySaml({ xmlString: signedWithTimestamp, publicCert: publicCert })).toBeTruthy();
-
-    const signedEnvelope = signEnvelope({ xml: signedWithTimestamp, privateKey });
-    expect(verifySaml({ xmlString: signedEnvelope, publicCert: publicCert })).toBeTruthy();
-  });
-
-  it("should sign and verify the ITI39 envelope successfully", () => {
-    const soapEnvelope = createITI39SoapEnvelope({
-      bodyData: iti39BodyData,
-      publicCert: publicCert,
-    });
-
-    const signedWithTimestamp = signTimestamp({ xml: soapEnvelope, privateKey });
-    expect(verifySaml({ xmlString: signedWithTimestamp, publicCert: publicCert })).toBeTruthy();
-
-    const signedEnvelope = signEnvelope({ xml: signedWithTimestamp, privateKey });
-    expect(verifySaml({ xmlString: signedEnvelope, publicCert: publicCert })).toBeTruthy();
-  });
-});
+export const IHE_STAGING_CERT = `-----BEGIN CERTIFICATE-----
```

Comment:
`IHE_STAGING_CERT` is not really our staging cert, right?

If not, any particular reason to name it like so? This is confusing :slightly_smiling_face: Could we name it with something else, like "mock", "valid"/'invalid"?

If yes, we should revoke it and generate a new one only for testing purposes that can make this public repo.

Reply:
This is a random cert from the internet. I will rename it :)